var/spack/repos/builtin/packages/dakota/package.py

```diff
@@ -64,8 +64,12 @@ class Dakota(CMakePackage):
 
     depends_on("python")
     depends_on("perl-data-dumper", type="build", when="@6.12:")
-    depends_on("boost@:1.68.0", when="@:6.12")
-    depends_on("boost@1.69.0:", when="@6.18:")
+    depends_on(
+        "boost@:1.68.0 +filesystem +program_options +regex +serialization +system", when="@:6.12"
+    )
+    depends_on(
+        "boost@1.69.0: +filesystem +program_options +regex +serialization +system", when="@6.18:"
+    )
```

Comment:
Can you add this as an unconditional:
```
depends_on("boost +filesystem +program_options +regex +serialization +system")
```
and leave the previous directives untouched? The assumption is that we might need to tweak the minimum version of `boost`, but we'll likely change the modules required much less.

Reply:
Done

Reply:
@alexrichert Still missing the suggestion below

Reply:
Okay sorry, I misunderstood what you meant by leaving those untouched. Committed the suggestion.